src/App/Resources/AppResources.resx

```diff
@@ -1880,11 +1880,17 @@
   <data name="TypeFileInfo" xml:space="preserve">
     <value>The file you want to send.</value>
   </data>
-  <data name="TypeIsSelected" xml:space="preserve">
-    <value>type is, selected.</value>
+  <data name="FileTypeIsSelected" xml:space="preserve">
+    <value>File type is, selected.</value>
   </data>
-  <data name="TypeIsNotSelected" xml:space="preserve">
-    <value>type is, not selected, click to select.</value>
+  <data name="FileTypeIsNotSelected" xml:space="preserve">
+    <value>File type is, not selected, click to select.</value>
+  </data>
+  <data name="TextTypeIsSelected" xml:space="preserve">
+    <value>Text type is, selected.</value>
+  </data>
+  <data name="TextTypeIsNotSelected" xml:space="preserve">
+    <value>Text type is, not selected, click to select.</value>
```

Comment:
Remove comma after "...type is," and change "click" to "tap" as in mobile the user taps a control, e.g. "File type is, not selected, click to select" should be "File type is not selected, tap to select"

Reply:
I added the comma because it makes the Voice Over be easier to understand, without it gets confusing because of the automatic robotic voice. Tap makes more sense though.

Reply:
The commas should be excluded.  TTS can vary wildly between engines & languages, and we can't rely on crowdsourcing to test & adjust every variation.  What sounds better in one situation could be disastrous in another. 
 It's the responsibility of the synthesizer to manage cadence & delivery.